app/src/main/java/com/edu/uni/augsburg/uniatron/ui/MainActivity.java

```diff
@@ -151,13 +171,6 @@ public void onNextClicked() {
         mModelNavigation.nextData();
     }
 
-    /**
-     * Is called when the fab is clicked.
-     */
-    @OnClick(R.id.fab)
-    public void onFabClicked() {
```

Comment:
Don't remove this method! Otherwise the shop will not be opened by pressing the floating action button